Doc/howto/regex.rst

```diff
@@ -844,7 +844,7 @@ backreferences in a RE.
 
 For example, the following RE detects doubled words in a string. ::
 
-   >>> p = re.compile(r'(\b\w+)\s+\1')
+   >>> p = re.compile(r'(\b\w+)\s+\1\b')
```

Comment:
I would move `\b` outside of the group: `r'\b(\w+)\s+\1\b'`. There is no need to keep it in the group, because it doesn't match any character. And this will add a symmetry in the expression.